### PR TITLE
Add GD32E103VB to platform database

### PR DESCRIPTION
**Pull request by ChazJin**

### CODE REVIEW

**Comment by Ronny-Liu** [comment]
@bridadan hi Brian, I dont see GD32_103V in the database.  The correct entry should be GD32_E103VB.  The change in PR is correct, though not reflected accurately in title.

**Comment by bridadan** [comment]
Hey @Ronny-Liu, I made a typo in my above comment, sorry about that! Let me try again 😄 

In the database, the  "logicalboard" for this board is listed as `GD32_E103V`. As you can see, this is missing the `B` on the end. Was this intentional?

**Comment by Ronny-Liu** [comment]
@bridadan ah, yes this is my bad.  I have fixed in our database. Thanks for pointing this out.
